File: app/src/main/java/me/sunhapper/spcharedittool/SpApplication.java
package me.sunhapper.spcharedittool;

import android.app.Application;
import me.sunhapper.spcharedittool.emoji.EmojiManager;

/**
 * Created by sunha on 2018/2/5 0005.
 */

public class SpApplication extends Application {

  @Override
  public void onCreate() {
    super.onCreate();
    EmojiManager.getInstance().init(this);
  }
}

File: app/src/main/java/me/sunhapper/spcharedittool/SpApplication.kt
package me.sunhapper.spcharedittool

import android.app.Application
import me.sunhapper.spcharedittool.emoji.EmojiManager

/**
 * Created by sunha on 2018/2/5 0005.
 */

class SpApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        EmojiManager.init(this)
    }
}
